env.yaml

```diff
@@ -3,16 +3,22 @@ MEMBRANE_CORS_ALLOWED_ORIGINS:
   field_info:
     default: http://localhost:3000
 
+MEMBRANE_SECRET_KEY:
+  type: str
+  optional: true
+
 MEMBRANE_FRONTEND:
   type: str
   field_info:
     default: http://localhost:3000
+    description: the membrane frontend
 
 MEMBRANE_SERVER_PRIVATE_KEY:
   type: FileStr
 
 MEMBRANE_SENDER_EMAIL:
   type: EmailStr
   field_info:
+    default: null
     pattern: ^[a-zA-Z0-9._+]+@(?:gc\.ca|canada\.ca|inspection\.gc\.ca)$
```

Comment:
1) what is I wanted to link to the RFC document URL detailing what makes an email valid in this env.yaml?
2) seems to me we should be configuring MEMBRANE_ALLOWED_EMAIL_DOMAINS instead and internally creating that regexp pattern